mopidy/internal/validation.py

```diff
@@ -31,16 +36,7 @@
     "musicbrainz_id",
 }
 
-DISTINCT_FIELDS = {
-    "track",
-    "artist",
-    "albumartist",
-    "album",
-    "composer",
-    "performer",
-    "date",
-    "genre",
-}
+DISTINCT_FIELDS = dict(TRACK_FIELDS_WITH_TYPES)
```

Comment:
Is creating a fresh new dict here intentional, or should it just be a direct assignment?

Reply:
I wanted the clear distinction compared to the search fields. And this happens at import time so it costs nothing.